### ISSUE TITLE
add support for array of middlewares in app.use

### ISSUE THREAD

**Issue by idangozlan** [malijs/mali#27]
I think we should add support for multiple middlewares as app.use param, for ex.

`app.use([sayHello, sayGoodbye]);`

**Comment by bojand** [comment]
Hmm not sure I understand exactly what you mean but this concept should already be supported. There was a slight bug that I just addressed in the `0.7.1` release. But for example you can do something like this:

```js
const path = require('path')
const Mali = require('mali')

const PROTO_PATH = path.resolve(__dirname, '../protos/helloworld.proto')
const HOSTPORT = '0.0.0.0:50051'

async function fn1 (ctx, next) {
  console.log('>> one')
  await next()
  console.log('<< one')
}

async function fn2 (ctx, next) {
  console.log('>> two')
  await next()
  console.log('<< two')
}

async function fn3 (ctx, next) {
  console.log('>> three')
  await next()
  console.log('<< three')
}

function sayHello (ctx) {
  console.log('got sayHello request')
  ctx.res = { message: 'Hello ' + ctx.req.name }
  console.log('set sayHello response to %j', ctx.res)
}

function main () {
  const app = new Mali(PROTO_PATH, 'Greeter')
  app.use(fn1, fn2, fn3)
  app.use({ sayHello })
  app.start(HOSTPORT)
  console.log(`Greeter service running @ ${HOSTPORT}`)
}

main()
```

When called:

```
Greeter service running @ 0.0.0.0:50051
>> one
>> two
>> three
got sayHello request
set sayHello response to {"message":"Hello world"}
<< three
<< two
<< one
```

**Comment by bojand** [comment]
Closing the issue as the current API supports this as I understand the issue. If something else is the issue or needed please reopen.
